CMakeLists.txt

```diff
@@ -380,13 +405,6 @@ set_property(TARGET freeorionparseobj
     POSITION_INDEPENDENT_CODE ON
 )
 
-if(WIN32)
```

Comment:
There is no need to reorder blocks. Also breaks exports.

Reply:
You mean the Parsers name should correspond to the project "freeorionparseobj"? 

I think this comes from a unsuccessful attempt to fix the CMake build. From my understanding, "Parsers" should correspond to the "freeorionparse"

Reply:
If you go to v0.4.8, this is how it's look like, but in the wrong place

`if(WIN32)
    set_property(TARGET freeorionparse
        PROPERTY
        OUTPUT_NAME Parsers
    )
endif()`